templates/corporate/pricing_model.html

```diff
@@ -86,7 +86,7 @@ <h2>Standard</h2>
                                     </div>
                                 </div>
                                 {% if is_cloud_realm and on_free_tier and not sponsorship_pending %}
-                                <a href="/upgrade/?tier={{ tier_cloud_standard }}" class="button upgrade-button">
+                                <a href="/upgrade/" class="button upgrade-button">
```

Comment:
Shouldn't this still pass `tier_cloud_standard`? I think it's better to be explicit.

Reply:
Okay, updated!